gorush/TAG

```diff
@@ -1 +1 @@
-emiksk-1.11.2.2
```

Comment:
gorush is not managed by us.  It is managed by Maneki.
Ignore it.

Reply:
We reverted.